docs/site/Examples.md

```diff
@@ -0,0 +1,29 @@
+---
+lang: en
+title: Example Projects
+keywords: LoopBack 4.0
+sidebar: lb4_sidebar
+permalink: /doc/en/lb4/Examples.html
+---
+
+LoopBack 4 comes with the following example projects:
+
+- **[rpc-server](https://github.com/strongloop/loopback-next/tree/master/examples/rpc-server)**:
+  An example showing how to implement a made-up RPC protocol.
+
```

Comment:
I think it should include `hello-world`.  See the discussion https://github.com/strongloop/loopback-next/pull/2327/files#r254569016.